src/components/Input-Form/Input-Form.js

```diff
@@ -0,0 +1,17 @@
+import './Input-Form.css';
+
+const InputForm = () => (
+  <form className="input-form">
+    <label htmlFor="title">
+      Title
+      <input type="text" name="title" id="title" required />
+    </label>
+    <label htmlFor="author">
+      Author
+      <input type="text" name="author" id="author" required />
+    </label>
+    <input type="submit" />
```

Comment:
- [ ] Please can we make this a button?. The requirements of this project stipulate that this should be a button.